components/blitz/resources/omero/System.ice

```diff
@@ -33,6 +30,9 @@ module omero {
         ["java:type:java.util.ArrayList<Integer>:java.util.List<Integer>"]
             sequence<int> IntList;
 
+        ["java:type:java.util.ArrayList<String>:java.util.List<String>"]
+            sequence<string> StringSet;
```

Comment:
That's unfortunate. Looks like I made some improvement on this once:

```
commit 830b799130acde2f92a908f59526cb8a84a07918
Author: jmoore <josh@glencoesoftware.com>
Date:   Fri Nov 18 15:27:53 2011 +0100

    Re-arrange System/Collections Ice imports

    3.4 for some reason is more strict on the order of imports.
    This is likely caused by package-based import cycles, but it's
    unclear.
```

Looks like the current cause is the need to have `EventContext` in Collections. One option would be a `SystemF.ice` like `ModelF.ice`. I can try to open a clean up PR on your branch after this is merged if you'd prefer.